wallet/wallet-cli-lib/src/commands/command_handler/mod.rs

```diff
@@ -375,6 +377,76 @@ where
                 Ok(ConsoleCommand::Print(addresses_table.to_string()))
             }
 
+            ColdWalletCommand::ShowStandaloneAddresses => {
+                let (wallet, selected_account) = wallet_and_selected_acc(&mut self.wallet).await?;
+                let addresses_with_label =
+                    wallet.get_standalone_addresses(selected_account).await?;
+
+                let addresses_table = {
+                    let mut addresses_table = prettytable::Table::new();
+                    addresses_table.set_titles(prettytable::row!["Address", "Type", "Label",]);
+
+                    addresses_table.extend(addresses_with_label.into_iter().map(|info| {
+                        let label = info.label.unwrap_or_default();
+                        let address_type = match info.address_type {
+                            AccountStandaloneKeyType::Address => "Address",
```

Comment:
"Address" is a bad name because Address means something already... this should probably maybe... "Unspecified"? I don't know...

Reply:
Added all types based on the destination